backend/coreapp/serializers.py

```diff
@@ -26,6 +26,7 @@ def serialize_profile(
             "is_anonymous": True,
             "id": profile.id,
             "is_online": profile.is_online(),
+            "username": f"{profile.pseudonym} (anon)" if profile.pseudonym else None,
```

Comment:
It's early and my brain is still warming up, but would this return pseudonyms for registered used instead of their GitHub username?

Reply:
This is in a 'if profile.user is None' block, so no